packages/frontend/src/app/modules/place/standalone-components/display-temp-banner/display-temp-banner.component.ts

```diff
@@ -72,6 +75,9 @@ export class DisplayTempBannerComponent
     this.dateString = this.dateService
       .translateDateInterval(this.tempInfos.dateDebut, this.tempInfos.dateFin)
       .toLowerCase();
+
+    this.shouldDisplay =
+      (this.forceDisplay && !!this.tempInfos.dateDebut) || this.tempInfos.actif;
```

Comment:
```suggestion
      (this.forceDisplay && Boolean(this.tempInfos.dateDebut)) || this.tempInfos.actif;
```

Reply:
D'ailleurs, ça serait pas plus juste d'avoir un truc comme ça :
```suggestion
      (this.forceDisplay || this.tempInfos.actif) && Boolean(this.tempInfos.dateDebut);
```
?